stable/database/files/nuosm

```diff
@@ -11,6 +11,7 @@
 
 : ${NUODB_ARCHIVEDIR:=/var/opt/nuodb/archive}
 : ${NUODB_BACKUPDIR:=/var/opt/nuodb/backup}
+: ${NUODB_JOURNALDIR:=/var/opt/nuodb/journal}
```

Comment:
Do we need to define our own default, rather than relying on the internal product default?

Reply:
this is what @acabrele set for the default. Opinions?

Reply:
Actually, I don't think this comment is relevant, since `$NUODB_JOURNALDIR` is only used if `SEPARATE_JOURNAL=true`.

Reply:
yeah. That is how I do the wiring. I do not allow the user to define the location of the journal. It always goes to `/var/opt/nuodb/journal/<DOMAIN>/<DBNAME>`. But of course only if `Values.database.sm.noHotCopy.journalPath.enable` is set to true